From c753c0cda1b443303c5c090da335a8a2f344ba4a Mon Sep 17 00:00:00 2001
From: "Chen, Tianmi" <tianmi.chen@intel.com>
Date: Mon, 11 Jul 2022 12:05:33 +0800
Subject: [PATCH] Enable DRM primary memory type in Android T

Enable DRM primary memory type in codec2.0

Tracked-On: OAM-101794
Signed-off-by: Chen, Tianmi <tianmi.chen@intel.com>
---
 mfx_c2_defs.mk | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/mfx_c2_defs.mk b/mfx_c2_defs.mk
index 9f2b1c1..580fff8 100644
--- a/mfx_c2_defs.mk
+++ b/mfx_c2_defs.mk
@@ -45,12 +45,7 @@ MFX_C2_CFLAGS += \
 
 ifeq ($(BOARD_USES_GRALLOC1),true)
   # plugins should use PRIME buffer descriptor since Android P
-  ifneq ($(filter MFX_S ,$(MFX_ANDROID_VERSION)),)
-      MFX_C2_CFLAGS += -DMFX_C2_USE_PRIME
-  endif
-  ifneq ($(filter MFX_R ,$(MFX_ANDROID_VERSION)),)
-      MFX_C2_CFLAGS += -DMFX_C2_USE_PRIME
-  endif
+  MFX_C2_CFLAGS += -DMFX_C2_USE_PRIME
 else
   $(error "Required GRALLOC1 support")
 endif
-- 
2.17.1

